spec/acceptance/nodesets/docker/CertsDockerfile

```diff
@@ -0,0 +1,67 @@
+FROM alpine:3.8
```

Comment:
We normally try to generate the dokerfiles on the fly by using https://github.com/puppetlabs/beaker-hostgenerator, but this one looks pretty specific, so a custom file is fine.